pylabnet/utils/helper_methods.py

```diff
@@ -96,4 +102,40 @@ def pwr_to_float(in_val):
     return str_to_float(in_val=in_val)
 
 
+def slugify(value, allow_unicode=False):
+    """
+    Convert to ASCII if 'allow_unicode' is False. Convert spaces to hyphens.
+    Remove characters that aren't alphanumerics, underscores, or hyphens.
+    Convert to lowercase. Also strip leading and trailing whitespace.
+
+    From Django 2.2
+    """
+    value = str(value)
+    if allow_unicode:
+        value = unicodedata.normalize('NFKC', value)
+    else:
+        value = unicodedata.normalize('NFKD', value).encode('ascii', 'ignore').decode('ascii')
+    value = re.sub(r'[^\w\s-]', '', value.lower()).strip()
+    return re.sub(r'[-\s]+', '-', value)
+
+
+def get_dated_subdirectory_filepath(directory, filename):
```

Comment:
Yes, the log ideally should be stored outside the pylabnet folder. This can be controlled by the script that instantiates the `LoggerService`. The following code for example creates the log folder in the same directory as the script (which should not be the pylabnet directory):

 
```python
log_service_name = 'G12 Logger'
log_folder = 'logs'

dir_path = os.path.join(os.path.abspath(''), log_folder)

log_service = LogService(
    log_output=True,
    name=log_service_name,
    dir_path=dir_path,
)
```

Even if log files are stored in the pylabnet directoy, the `.log` ending is included in the `.gitignore`, so they won't get commited (neither will the dated sub-directory structure). Not sure if we should implement an additional check, let's discuss.

Reply:
great!